### PR TITLE
Export Merger and merge function

### PR DESCRIPTION
**Pull request by vinceau**
Sometimes you just need the flattened source code as a string and not written to a file. This PR exports a `merge` function to be imported within node. The flattened source code as a string is returned.

We also export `Merger` to allow for more configurability if necessary.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+let Merger = require('./lib/merger');
+
+let merge = (file) => {
```
Comment (RyuuGan) [inline index.js]: This function can be simplified:

```suggestion
const merge = async (file) => {
  const merger = new Merger({ delimeter: '\n\n' });
  return merger.processFile(file, true);
}
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        try {
+            result = await merger.processFile(file, true);
+        } catch (e) {
+            reject(e);
+        }
+        resolve(result);
+    });
+};
+
+module.exports = merge;
```
Comment (RyuuGan) [inline index.js]: We should also export `Merger` then if you need some more options to set up.

```suggestion
module.exports.Merger = Merger;
module.exports.merge = merge;
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,18 @@
 ### Build status
 [![Build status](https://travis-ci.org/RyuuGan/sol-merger.svg?branch=master)](https://travis-ci.org/RyuuGan/sol-merger)
 
-### USAGE
+### Quick Usage
+
+```javascript
+const merge = require('sol-merger');
```
Comment (RyuuGan) [inline README.md]: This has to be changed due to last exports:

```suggestion
const { merge } = require('sol-merger');
```

**Review by RyuuGan — CHANGES_REQUESTED** [review]
Since merge is just a helper function we should also export `Merger` itself. 

Can you also add tests to `merge` function, like it is done at the second test:

https://github.com/RyuuGan/sol-merger/blob/master/test/index.js#L15

**Comment by vinceau** [comment]
The requested changes have been made and a test has been added for `merge`.

**Review by RyuuGan — APPROVED** [review]

**Comment by RyuuGan** [comment]
Merged and published. Available since version `0.1.4`.

**Pull request merged**